apps/artwork/components/auction/test/view.coffee

```diff
@@ -111,6 +112,25 @@ describe 'auction', ->
         @view.$('.artwork-auction__buy-now')
           .should.have.lengthOf 0
 
+    describe 'reserve not met', ->
+      it 'gives a reserve not met bid message', ->
+        data = Object.assign({}, @data)
```

Comment:
Nice cloning action! Since this gets re-created in the `beforeEach` it's probably overkill, but no harm no fowl. I like the thought!